src/system/Table/Table.stories.jsx

```diff
@@ -36,6 +36,34 @@ export const Default = () => (
 					</Text>,
 				] }
 			/>
+			<TableRow>
+				<TableCell sx={ { backgroundColor: 'gray' } }>
```

Comment:
```suggestion
				<TableCell sx={ { backgroundColor: 'lightgray' } }>
```
Moving it to `lightgray` still allows this to be visible and doesn't raise an A11Y violation in the storybook

Reply:
Awesome! I didn't notice the violation. Thank you